### ISSUE TITLE
The output at the bottom of the page in incorrect, 

### ISSUE THREAD

**Issue by SuperCool83** [dotnet/docs#5121]
in the example provided in the MyAssembly.fs and file1.fsx
x = 10
y = 40
so 10 + 2 x 40 = 90.
not 60.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3ea5b069-0d43-f577-3f95-fde5aaf919d5
* Version Independent ID: 0ea5685e-45cc-06d2-3985-cffb530f9e67
* Content: [F# Interactive (fsi.exe) Reference](https://docs.microsoft.com/en-us/dotnet/fsharp/tutorials/fsharp-interactive/)
* Content Source: [docs/fsharp/tutorials/fsharp-interactive/index.md](https://github.com/dotnet/docs/blob/master/docs/fsharp/tutorials/fsharp-interactive/index.md)
* Service: **unspecified**
* GitHub Login: @cartermp
* Microsoft Alias: **phcart**

**Comment by svick** [comment]
You're right, the page should say that the output is `90`. Do you want to create a Pull Request that fixes it? (If you do, but want some help with that, feel free to ask.)

**Comment by SuperCool83** [comment]
I would love to create a pull request to fix it and do need help with that.

What steps do I take?


Yoni Higgsmith



+44 (0) 7916 124855 <%2B44%280%297916%20124855>
 elphilosopher
Twitter: @16x16table

**Comment by svick** [comment]
The easiest approach is:

* Go to [the page you want to edit](https://docs.microsoft.com/en-us/dotnet/fsharp/tutorials/fsharp-interactive/).
* Click the Edit button to the upper right corner. This will take you to the source for the page on GitHub.
* Click on the pencil icon in the upper right corner of the document. This will actually take you to a page where you will be able to edit the page.
* Make your change, optionally set the commit message to something better than "Update index.md" and click Propose file change.
* Click Create pull request.
* Change the Pull Request description to "Fixes #5121" to link it back to this issue, then click Create pull request.

**Comment by SuperCool83** [comment]
I worked it out but didn't know about linking back to this issue.   
Thank you for your help

**Comment by pkulikov** [comment]
@svick what you've just done is super cool.

**Comment by mairaw** [comment]
You all rock! @BillWagner have you published that one pager with the quick instructions for contributing to docs?
